### PR TITLE
Stanag4559 temporal/spatial

### PR DESCRIPTION
**Pull request by bdeining**
@kcwire 
@jrnorth 
@dcruver 
@jaymcnallie 
@shane-voisard 
@jlcsmith 
@troymohl

### CODE REVIEW

**Comment by bdeining** [comment]
#3 is built on top of this PR. This will be kept open for code review. Recommend not merging this, and merging that PR instead.

File: catalog/nato/stanag4559/catalog-stanag4559-source/src/main/java/alliance/catalog/nato/stanag4559/source/Stanag4559FilterDelegate.java [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package alliance.catalog.nato.stanag4559.source;
```
Comment (jlcsmith) [inline catalog/nato/stanag4559/catalog-stanag4559-source/src/main/java/alliance/catalog/nato/stanag4559/source/Stanag4559FilterDelegate.java]: we should discuss package names.  I think we should have ddf in the name.

File: catalog/nato/stanag4559/catalog-stanag4559-source/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        <AD description="The Username to be used for authentication with HTTP server."
+            name="HTTP(s) Username" id="cxfUsername"
+            required="false" type="String"/>
+
+        <AD description="The Password to be used for authentication with HTTP server."
+            name="HTTP(s) Password" id="cxfPassword"
+
+            required="false" type="String"/>
+
+        <AD description="The Key to be used in Access Criteria" name="Key" id="key" required="false"
```
Comment (jlcsmith) [inline catalog/nato/stanag4559/catalog-stanag4559-source/src/main/resources/OSGI-INF/metatype/metatype.xml]: what is this key?

File: catalog/nato/stanag4559/catalog-stanag4559-source/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            type="String"/>
+
+        <AD description="The Maximum Hit Count for Queries to the Source" name="Max Hit Count"
+            id="maxHitCount" required="true" type="Integer" default="250"/>
+
+        <AD description="Poll Interval to Check if the Source is available (in minutes - minimum 1)."
+            name="Poll Interval" id="pollInterval"
+            required="true" type="Integer" default="20"/>
```
Comment (jlcsmith) [inline catalog/nato/stanag4559/catalog-stanag4559-source/src/main/resources/OSGI-INF/metatype/metatype.xml]: 20 minutes seems like a lot.  Any reason to not check more frequently?

Comment (bdeining) [inline catalog/nato/stanag4559/catalog-stanag4559-source/src/main/resources/OSGI-INF/metatype/metatype.xml]: Reduced to 5 minutes.

File: catalog/nato/stanag4559/catalog-stanag4559-source/src/main/resources/OSGI-INF/metatype/metatype.xml (hunk 2 above)
Comment (bdeining) [inline catalog/nato/stanag4559/catalog-stanag4559-source/src/main/resources/OSGI-INF/metatype/metatype.xml]: The key is part of the access criteria, which is actually defined and not used.  The configurable properties ( key, access criteria username and password) will be removed.

File: catalog/nato/stanag4559/catalog-stanag4559-source/src/main/java/alliance/catalog/nato/stanag4559/source/Stanag4559FilterDelegate.java (hunk 1 above)
Comment (bdeining) [inline catalog/nato/stanag4559/catalog-stanag4559-source/src/main/java/alliance/catalog/nato/stanag4559/source/Stanag4559FilterDelegate.java]: As discussed, package name will be changed to org.codice.alliance.appname.bundlename.classname

**Comment by bdeining** [comment]
https://github.com/codice/alliance/pull/6.  Closing this PR and breaking up the code to make it easier to review.  All comments were addressed.

**Pull request closed without merging**